app/injectors/import_rdf_injector.rb

```diff
@@ -0,0 +1,42 @@
+# Injector for various RDF importing dependencies
+class ImportRdfInjector
+  def url_to_graph
+    ->(url) { RdfFetcher.call(url) }
```

Comment:
Haha, yup.  I think it was previously wrapping something else, but it kept getting reworked until it ended with the absurdity you saw.